### ISSUE TITLE
Incense categories

### ISSUE THREAD

**Issue by SamsaSpoon** [perfidiousscript/incense_app#11]
I think there should be categories like Coreless - Masala - Dipped - Dhoop.

**Comment by perfidiousscript** [comment]
This is a good idea. Just to be clear, this would be something that would be selected when creating a new incense(?)

**Comment by SamsaSpoon** [comment]
This would make the most sense. 
I'm just not sure how detailed these categories should be. "Coreless" could be split to Japanese style, Bhutanese/Tibetan, Chinese joss sticks - and technically Dhoop sticks are coreless too but soft and kneadable (as far as I know). 
And there is soft and dry masala...

**Comment by perfidiousscript** [comment]
Yeah this is one of the big problems around these sorts of things, trying to strike the balance between too general and too specific.

I'm not super familiar with these different forms, but I figure if you can come up with 4-6 of the most common kinds (probably ignoring subcategories) I could put those in and it would cover 80% of commercially made incense. Of course this can always be refined, added too et c. down the line.

**Comment by SamsaSpoon** [comment]
I'll think about it and get back to you.

**Comment by SamsaSpoon** [comment]
Here are my thoughts: 
I'd make two columns; One with "Style", the other "Shape". 
Styles are: Coreless - Japanese; Coreless - Tibetan/Bhutanese; Masala; Dipped/Flora; Dhoop; Other
Shapes are: Cone; Coil; Other
One can choose one per column. If there is no Shape chosen it defaults to Stick. Maybe there should be a default unknown if no Type is chosen. 
If you think it gets to specific with the Shape column, it can be left out. But I think that's a good way to cover all Types of incense that is "light up and go" (In contrast to for example loose or kneaded incense that neads a seperate heat source to work.)
